app/commands/fever_api/read_feeds_groups.rb

```diff
@@ -13,9 +13,11 @@ def call(authorization:, **params)
     private
 
     def feeds_groups(authorization)
-      scoped_feeds = authorization.scope(FeedRepository.in_group)
+      scoped_feeds = authorization.scope(FeedRepository.list)
       grouped_feeds = scoped_feeds.order("LOWER(name)").group_by(&:group_id)
       grouped_feeds.map do |group_id, feeds|
+        group_id ||= Group::UNGROUPED.id
```

Comment:
I haven't tried to be honest. Given we're dealing with a Java (Kotlin) client here, where `null` is not a proper object but crops up everywhere, I was playing it safe with using `0`. 

If you want to try to trace the client code, the juicy bits are here: 
* https://github.com/Ashinch/ReadYou/blob/main/app/src/main/java/me/ash/reader/domain/service/FeverRssService.kt#L159-L168
* https://github.com/Ashinch/ReadYou/blob/main/app/src/main/java/me/ash/reader/infrastructure/rss/provider/fever/FeverDTO.kt

Reply:
Hmm, well I haven't worked with Java in 15 years so I can't be sure, but [seeing `?`](https://github.com/Ashinch/ReadYou/blob/441368695cffd992181f08184139bbdc792cf346/app/src/main/java/me/ash/reader/infrastructure/rss/provider/fever/FeverDTO.kt#L90C9-L90C28) suggests to me that the group id is optional. Would you be up for trying it with `nil`? We can always follow up and make it something else if that doesn't work.

Reply:
I'll give it a shot.

Notice the `?` is a Kotlin feature, not a Java one. Kotlin extends the type system with these `?` types, the `?` are part of the type signature. For each type there's an implicit `?` or nullable type. For calling Java libraries all Java defined type signatures are the nullable type in Kotlin. Kotlin will statically prevent passing null to a non-nullable typed parameter or variable and it will statically force you to check for null when moving from nullable to non-nullable. However null is still the absence of a value and doesn't become a singleton object like `nil` is in Ruby. So it's still no good to use for hash map keys and stuff like that.

Reply:
So quite as I expected, no luck with `nil` as group ID. I fully expected the app to crash to be honest, which it didn't. It doesn't show any error either however, the list of groups and stories simply stays empty and there's even less requests.

Reply:
Alright, thanks for trying it out!